test/integration/test-amp-ad-doubleclick.js

```diff
@@ -29,20 +29,17 @@ describe('Rendering of one ad', () => {
   });
 
   it('ad should create an iframe loaded', function() {
-    this.timeout(5000);
+    this.timeout(10000);
```

Comment:
In practice this succeeds reliably, but each steps can take long in itself. Just increased the total, so when everything is slow we don't time out the test itself.